packages/components/src/components/popover/style.scss

```diff
@@ -1,5 +1,6 @@
 @import '../@shared/mixins';
-@import '../style';
+
+@layer kol-global, kol-component, kol-theme-global, kol-theme-component;
```

Comment:
Das hier wäre streng genommen nicht notwendig, wenn die Module (=Layers) in der richtigen Reihenfolge eingefunden werden. Können wir aber von meiner Seite aus gerne auch explizit definieren.
Sollten wir das dann jedoch nicht global bzw. für jede Komponente machen?

Reply:
Bei den beiden Komponenten popover, tooltip, die eigenen Styles als WC hatten, weg genau die Reihenfolge ein Problem.

Jetzt haben Komponenten keinen eigenen Style mehr und deshalb versuchen wir es mal wieder zu entfernen.